docs/guides/composing-messages.md

```diff
@@ -69,4 +69,147 @@ ChatPostMessageResponse response = slack.methods(token).chatPostMessage(req -> r
 );
 ```
 
-You can use this way also for building a message for Incoming Webhooks and `response_url` calls.
\ No newline at end of file
+You can use this way also for building a message for Incoming Webhooks and `response_url` calls.
+
+---
+## Block Kit Kotlin DSL
+
+If you're using Kotlin, the Java Slack SDK also provides Kotlin-style builders which you can use to build your block kit structures.
+
+```kotlin
+import com.slack.api.model.block.Blocks.*
+import com.slack.api.model.kotlin_extension.request.chat.blocks
+
+val response = slack.methods(token).chatPostMessage { req -> req
```

Comment:
Just in case, let's have the lines of code initializing the `Slack` instance and the token (also necessary imports) in the beginning.

```kotlin
val slack = Slack.getInstance()
val token = System.getenv("SLACK_TOKEN")
```